### ISSUE TITLE
Adding new exchanges

### ISSUE THREAD

**Issue by Tucsky** [Tucsky/SignificantTrades#1]
If you want a exchange to be added into this tracker, this is the thread.
Answer here with the **name of the plateform** and **why you think it coult be relevant here**.

I previously added bithumb and it has been found very unstable in term of **price action** and **technicaly speaking**. So you might want to take two things needs into account.

**Comment by Sponge001** [comment]
Hi tucsky, could you please add Okex Futures to the list of exchanges? Currently it only supports the spot exchange on Okex.

Thank you very much for this amazing peice of software!

**Comment by Tucsky** [comment]
Hello Sponge001 :)
Yeah there is only spot exchanges since the goal of this app is to aggregate many exchanges and see live fluctuation on relatively similar markets.
So how do you expect to use futures inside this app ? Im wondering... the way I see it is that you would enter like "BTC0727" and be tracking that okex market but thats not really useful imo, easy to implement tho :)
I don't do futures so I probably missing your point

**Comment by Sponge001** [comment]
Hi Tucsky,

I'm probably not using the software like your original intention. I often have several instances of it running and I break it down into several exchanges to get a gauge of which exchange is moving the most at the current time. I have recently moved to trading Okex futures. They move somewhat different to other exchanges so being able to see the difference is very valuable to me. I usually trade the quarterly future (btc0928 currently) as it has more liquidity. 

This really is a great piece of software! I would really appreciate if you could add Okex futures. I have sent you an email to the email address on your website, I can provide you more information on how i use this software, but I cannot post it publicly.

Thanks

Hamish

**Comment by Sponge001** [comment]
Hi Tucsky,

I'm probably not using the software like your original intention. I often have several instances of it running and I break it down into several exchanges to get a gauge of which exchange is moving the most at the current time. I have recently moved to trading Okex futures. They move somewhat different to other exchanges so being able to see the difference is very valuable to me. I usually trade the quarterly future (btc0928 currently) as it has more liquidity.

This really is a great piece of software! I would really appreciate if you could add Okex futures. I have sent you an email to the email address on your website, I can provide you more information on how i use this software, but I cannot post it publicly.

Thanks

Sponge001

**Comment by Sponge001** [comment]
Hi Tucsky,
Can you please confirm your btc wallet address, I would like to send a donation.

Thanks
Sponge001

**Comment by Tucsky** [comment]
Sorry dude, been out for the weekend with no access to code..

So I just added OKEX Futures Contracts just now on the *current release*,
simple update tho, just type BTC1WEEK to point on track weekly contracts,
BTC2WEEKS for bi-weekly & BTCQUARTER
(BTC, LTC, ETH, ETC, BCH, XRP, EOS, BTG)

Better implementation coming in the next release which I need to focus
(starting with real contracts name ofc)

Thanks again for your feedbacks, much appreciated ✌️

Le ven. 27 juil. 2018 à 01:39, Sponge001 <notifications@github.com> a
écrit :

**Comment by Sponge001** [comment]
Thanks man

**Comment by SubNoizey** [comment]
Liquid exchange - https://coinmarketcap.com/exchanges/liquid/
Not sure If you can add BTC/JPY but they're a top 5 exchange globally (usually sitting 2nd or 3rd) and the largest exchange in Japan which you can see below.

https://coinmarketcap.com/currencies/bitcoin/#markets

The USD market will continue to grow as they expand globally.

USD Market - https://app.liquid.com/margin/BTCUSD

**Comment by Tucsky** [comment]
Thanks I'll check it out!

**Comment by ciekawy** [comment]
maybe cctx? https://github.com/ccxt/ccxt

**Comment by Tucsky** [comment]
Idk sir last time I checked was about 6 months ago, ccxt wasn't supporting websocket at all

**Comment by quantfiction** [comment]
Deribit and Bybit both offer perpetual swap products that are becoming increasingly liquid.. might be interesting to be able to compare to Bitmex/OKEX

**Comment by Tucsky** [comment]
@quantfiction Indeed, I was planning to add deribit but didn't published the code yet
Do you know why Deribit isn't listed on cryptomarketcap ?
Can't find Bybit either...

**Comment by quantfiction** [comment]
RE: why those exchanges aren't on CMC, I'm really not sure.. since they trade primarily/exclusively derivatives contracts, they not be considered a "true" exchange. But they have legitimate volume as far as I'm aware and significant trades done on both sites should be be considered truly "significant" imo

**Comment by Tucsky** [comment]
Done adding thoses two, Deribit data looks good  :), added it on the old app too.
Bybit is a little early to say, very new api, subscribing to the trade feed gives u all the pairs trades (not a lot on bybit so its fine I guess). I disabled it by default.

**Comment by quantfiction** [comment]
I think you may have enabled *only* bybit.. that's all I see when I re-load the page

**Comment by ciekawy** [comment]
@Tucsky from what I saw there is quite advanced progress on WS implementation - it may be worth tracking
https://github.com/ccxt/ccxt/issues/56

https://github.com/lfern/ccxt/blob/feature/websockets-multiple/WEBSOCKETS.md

**Comment by hanyoonLDA** [comment]
Have you checked out https://github.com/altangent/ccxws?

**Comment by zhu1040028623** [comment]
Hi, could you add BitCoke exchange to your tracker. Website: https://www.bitcoke.com/
For two reasons: 1. BitCoke has very active trading volume in Asia market.  2. It's extremely stable without downtime. On March 12&13 Bitcoin price slump, BitCoke withstood the extreme volatility, no erratic price action.
